contracts/PartyBid.sol

```diff
@@ -28,6 +28,14 @@ contract PartyBid is Party {
     //   (1) PartyStatus.ACTIVE on deploy
     //   (2) PartyStatus.WON or PartyStatus.LOST on finalize()
 
+    enum ExpireCapability {
+        CanExpire, // The party can be expired
+        BeforeExpiration, // The expiration date is in the future
+        PartyInactive, // The party is inactive, either already expired, won, or lost.
```

Comment:
nit: "already expired" and "lost" is the same thing. I would maybe rename this to "PartyOver" so it's more clear